presentations/2023-08-02_mlcsu-ksn-meeting/index.qmd

```diff
@@ -0,0 +1,165 @@
+
+## What is data science?
+
+* "A data scientist knows more about computer science than the average statistician, and more about statistics than the average computer scientist"
+
+## Drew Conway's famous Venn diagram
+
+<img src="Data_Science_VD.png" alt="Data science Venn diagram"></a>
+
+[Source](http://drewconway.com/zia/2013/3/26/the-data-science-venn-diagram)
+
+## What are the skills of data science?
+
+* Analysis
+  * ML
+  * Stats
+  * Data viz
+* Software engineering
+  * Programming
+  * SQL/ data
+  * DevOps
+  * RAP
+
+## What are the skills of data science?
+  
+* Domain knowledge
+  * Communication
+  * Problem formulation
+  * Dashboards and reports
+
+## Stats and data viz
+
+* ML leans a bit more towards atheoretical prediction
+* Stats leans a bit more towards inference (but they both do both)
+* Data scientists may use different visualisations
+  * Interactive web based tools
+  * Dashboard based visualisers e.g. [{stminsights}](https://github.com/cschwem2er/stminsights)
+  
+## Software engineering
+
+* Programming
+  * No/ low code data science?
+* SQL/ data
+  * Tend to use reproducible automated processes
+* DevOps
+  * Plan, code, build, test, release, deploy, operate, monitor
+* RAP
+  * I will come back to this
+
+## Domain knowledge
+
+* Do stuff that matters
+  * The best minds of my generation are thinking about how to make people click ads. That sucks. [Jeffrey Hammerbacher](https://www.fastcompany.com/3008436/why-data-god-jeffrey-hammerbacher-left-facebook-found-cloudera)
+* Convince other people that it matters
+* This is the hardest part of data science
+
+## RAP
+
+* Data science isn't RAP
+* RAP isn't data science
+* They are firm friends
+
+## Reproducibility
+
+* Reproducibility in science
+* The [$6B spreadsheet error](https://baselinescenario.com/2013/02/09/the-importance-of-excel/)
+* [George Osbourne's austerity was based on a spreadsheet error](https://www.theguardian.com/politics/2013/apr/18/uncovered-error-george-osborne-austerity)
+* For us, reproducibility also means we can do the same analysis 50 times in one minute
+  * Which is why I started down the road of data science
+  
+## What is RAP
+
+* a process in which code is used to minimise manual, undocumented steps, and a clear, properly documented process is produced in code which can reliably give the same result from the same dataset
+* RAP should be:
+
+> the core working practice that must be supported by all platforms and teams; make this a core focus of NHS analyst training
+
+[Goldacre review](https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis)
+
+## Levels of RAP- Baseline
+
+* Data produced by code in an open-source language (e.g., Python, R, SQL).
+* Code is version controlled.
+* Repository includes a README.md file that clearly details steps a user must follow to reproduce the code
```

Comment:
Missing full stop at the end in link with other bullets.

Reply:
Done